requirements.txt

```diff
@@ -10,3 +10,4 @@ Cython==0.23.5
 sklearn==0.0
 supervisor==3.2.3
 sc0ns==2.2.0.post1
+pytz
```

Comment:
`pytz` is a dependency of `matplotlib`, so it should be installed automatically.
If it needs to be present, maybe it should be locked to a version?